tests/RestHandler/RestViewHandlerTest.cpp

```diff
@@ -58,17 +58,17 @@ struct TestView : public arangodb::LogicalView {
   TestView(TRI_vocbase_t& vocbase,
            arangodb::velocypack::Slice const& definition)
       : arangodb::LogicalView(*this, vocbase, definition) {}
-  virtual arangodb::Result appendVelocyPackImpl(
-      arangodb::velocypack::Builder& builder, Serialization) const override {
+  arangodb::Result appendVPackImpl(arangodb::velocypack::Builder& builder,
```

Comment:
Out of curiosity, why most of the "builder" were renamed to "build" but not this one? Is there any "rule" I`m missing to grasp?

Reply:
Sorry for this changes, i renamed to build because its better formatted in some cases. Maybe I shouldn't

Reply:
This hasn't been renamed due to inattention